src/main/cpp/test/TPImpeller.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <subsys/Impeller.h>
+#include <controllers/impeller/ImpellerAgitate.h>
+#include <controllers/impeller/ImpellerOff.h>
+#include <controllers/impeller/ImpellerOn.h>
+#include <controllers/impeller/ImpellerStateMgr.h>
+#include <frc/Timer.h>
+
+class TPImpeller : ImpellerStateMgr
```

Comment:
should be a stand along class not a subclass

This will require that you have the state manger as an attribute (and include it) so that you can call it every loop